python/tests/test_dataframe.py

```diff
@@ -0,0 +1,919 @@
+#
+#  Copyright 2017 TWO SIGMA OPEN SOURCE, LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+import unittest
```

Comment:
Chatted with @leifwalsh 

Can you move 

test_dataframe.py
test_data.py
base_test_case.py
spark_test_case.py

back to tests/ts/flint? You might need to add __init__.py files